tests/phpunit/tests/option/wpLoadAlloptions.php

```diff
@@ -19,6 +19,28 @@ public function test_if_alloptions_is_cached() {
 		$this->assertNotEmpty( wp_cache_get( 'alloptions', 'options' ) );
 	}
 
+	/**
+	 * @covers ::wp_load_alloptions
+	 */
+	public function test_default_and_yes() {
+		add_option( 'foo', 'bar', '', 'default-yes' );
+		add_option( 'bar', 'foo', '', 'yes' );
+		$alloptions = wp_load_alloptions();
+		$this->assertArrayHasKey( 'foo', $alloptions );
+		$this->assertArrayHasKey( 'bar', $alloptions );
+	}
+
+	/**
+	 * @covers ::wp_load_alloptions
+	 */
```

Comment:
Please remember to include the `@ticket` annotations.
```suggestion
	/**
	 * @ticket 42441
	 *
	 * @covers ::wp_load_alloptions
	 */
	public function test_default_and_yes() {
		add_option( 'foo', 'bar', '', 'default-yes' );
		add_option( 'bar', 'foo', '', 'yes' );
		$alloptions = wp_load_alloptions();
		$this->assertArrayHasKey( 'foo', $alloptions );
		$this->assertArrayHasKey( 'bar', $alloptions );
	}

	/**
	 * @ticket 42441
	 *
	 * @covers ::wp_load_alloptions
	 */
```